src/components/DateSlider/index.js

```diff
@@ -131,10 +121,7 @@ const DateSlider = () => {
   useEffect(() => {
     if (!(countryContext === null || countryContext === '')) {
       availableDate.getDate(countryContext, species).then(date => {
-        const dateData = date.data
-
-        const dateFormat =
-          dateData.length > 0 ? dateMarksFormat(dateData) : dateData
+        const dateFormat = dateMarksFormat(date.data)
```

Comment:
We lost the ternary -- what happens when `date.data` has length 0 or is null? Does `dateMarksFormat` protect against that?